### ISSUE TITLE
implement one method (RISE/LIME) for images

### ISSUE THREAD

**Issue by cwmeijer** [dianna-ai/dianna-exploration#110]

**Comment by loostrum** [comment]
We chose RISE because it is more straightforward to implement than LIME.

**Comment by cwmeijer** [comment]
We implemented a version of RISE in the prototype branch in DIANNA (https://github.com/dianna-ai/dianna/tree/prototype).
We want to auto check the axes of the data to see if the channels axis is the second or the last axis.

**Comment by cwmeijer** [comment]
We'll find some solution to the multiple possible axes orders (pytorch vs onnx/keras) in some other issue.
